libs/elements/layout/src/lib/components/header/header.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="navbar">
+    <header class="header">
+            <div class="navbar-header">
+                elewa group
+            </div>
+            <ul class="navlinks">
+                <li><a href="#">Activities</a></li>
+                <li><a href="#">About us</a></li>
+                <li><a href="#">Social impact</a></li>
+                <li><a href="#">Invest</a></li>
+                <li><a href="#">News</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+          </header>
+      <hr style="width:92.5%;text-align:center;margin-bottom: 30px;">
+  </div>
+  
```

Comment:
This header still needs work, check the issue description again. Especially on the mobile design